custom_components/frigate/camera.py

```diff
@@ -279,6 +284,79 @@ async def favorite_event(self, event_id: str, favorite: bool) -> None:
         await self._client.async_retain(event_id, favorite)
 
 
+class BirdseyeCamera(FrigateEntity, Camera):  # type: ignore[misc]
+    """Representation of the Frigate birdseye camera."""
+
+    # sets the entity name to same as device name ex: camera.front_doorbell
+    _attr_name = None
+
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        frigate_client: FrigateApiClient,
+    ) -> None:
+        """Initialize the birdseye camera."""
+        self._client = frigate_client
+        FrigateEntity.__init__(self, config_entry)
+        Camera.__init__(self)
+        self._url = config_entry.data[CONF_URL]
+        self._attr_is_on = True
+        # The device_class is used to filter out regular camera entities
+        # from motion camera entities on selectors
+        self._attr_device_class = DEVICE_CLASS_CAMERA
+        self._attr_is_streaming = True
+        self._attr_is_recording = False
+        self._stream_source = f"rtsp://{URL(self._url).host}:8554/birdseye"
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID to use for this entity."""
+        return get_frigate_entity_unique_id(
+            self._config_entry.entry_id,
+            "camera",
+            "birdseye",
+        )
+
+    @property
+    def device_info(self) -> dict[str, Any]:
+        """Return the device information."""
+        return {
+            "identifiers": {
+                get_frigate_device_identifier(self._config_entry, "birdseye")
```

Comment:
This makes `birdseye` a device, as well as a camera entity associated with that device. Will there be other entities we think are likely to belong to this device?

[The alternative would be to put it into the existing `Frigate` device that we've already put a few common entities in, e.g. detection FPS, server version, etc]

No strong opinion, just flagging that as an option.

Reply:
Probably not for 0.12 but definitely in the future there may be switches or other platforms to toggle the birdseye behavior like which cameras are included or the current mode (objects, continuous, etc.)

Reply:
Sounds great. Thanks.